security/openpgpsdk/Portfile

```diff
@@ -21,7 +21,7 @@ checksums           md5     a2fe87ee864e6bcc0d5ea3db20e0f57a \
                     sha1    faa2eb844da0f1689dee29c3220364545ca7746b \
                     rmd160  9260e464f8f9b385d0cc8348d20e1c264856cee3
 
-depends_build       bin:perl:perl5 bin:gpg:gnupg
+depends_build       bin:perl:perl5 path:bin/gpg:gnupg-stable
```

Comment:
Why do you use path: dependencies. This basically disallows installing your own version of gnupg outside of macports.

Reply:
No, users can still install GnuPG outside of MacPorts - MacPorts will just not use it. And that's exactly the point. MacPorts strives to provide a self-contained system, save for the base system shipped by Apple. Since Apple does not ship GnuPG, these dependencies should never have been `bin:`-style dependencies in the first place. We do want to pull in our own version, over which we have control. Stuff that users install is outside of our control. We don't know what version that might be, we have no idea how it might have been patched, we don't even know if that's **really** GnuPG.